charts/seed-controlplane/charts/kube-controller-manager/templates/mcm-vpa.yaml

```diff
@@ -0,0 +1,14 @@
+{{ if .Values.vpa.enabled }}
```

Comment:
Rename file to kube-controller-manager-vpa.yaml

Reply:
Fixed